src/t8_schemes/t8_default/t8_default_prism/t8_dprism_bits.c

```diff
@@ -621,7 +621,7 @@ t8_dprism_linear_id (const t8_dprism_t *p, int level)
     /*The number to add to the id computed via the tri_id is 4*8^(level-i)
      *for each plane in a prism of size i*/
     id += line_id / line_level * prism_shift;
-    line_id = (uint64_t) (line_id % line_level);
+    line_id = (t8_linearidx_t) (line_id % line_level);
```

Comment:
Is casting necessary here at all? `line_id` and `line_level` are already `t8_linearidx_t`.

Reply:
You are right. I changed the name of ```line_level``` to ```lines_at_level```, because the name is misleading. ```line_level``` indicates that the variable stores a level, hence an integer. Which it doesn't! Thank you for catching that.